save.html

```diff
@@ -20,8 +20,23 @@ <h1>This is the Cocktail Finder saved recipes page</h1>
             <!--js rendered drink options render here-->
         </section>
         <section id="recipeRender">
+        <!-- Delete button for each recipes-->
+        <div class="flex-col max-w-[25%] min-w-[25%]">
+            <section class="flex justify-evenly"><!--Random recipe generator button-->
+                <button type="button" id="Delete Recipe"
+                    class="text-white bg-gradient-to-br from-purple-600 to-blue-500 hover:bg-gradient-to-bl focus:ring-4 focus:outline-none focus:ring-blue-300 dark:focus:ring-blue-800 font-medium rounded-lg text-sm px-5 py-2.5 text-center me-2 mb-2">Random
+                    Search</button>
+        <button>
+           
+        </button>
+        </select>
+
 
         </section>
+        <section>
```

Comment:
This also left an odd box on the page. I don't think we will need a multiple selection on the save page though.